StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+PaymentMethodAvailability.swift

```diff
@@ -43,10 +43,10 @@ extension PaymentSheet {
     ]
 
     /// A list of `STPPaymentMethodType` that can be saved in PaymentSheet
-    static let supportedSavedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit]
+    static let supportedSavedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit, .link]
```

Comment:
How does this work when CustomerSessions isn't set up (with the old ephemeral key flow) when we don't have the PM details? Do we just show the generic Link PMs, or are they not fetched at all?

Reply:
If a merchant is using ephemeral keys, then we call `/payment_methods` for all payment method types in `PaymentSheet.supportedSavedPaymentMethods`, which doesn't include Link.